content/en/observability_pipelines/guide/configure_observability_pipelines_with_datadog.md

```diff
@@ -1,80 +1,78 @@
 ---
-title: Integrate Vector with Datadog
+title: Configure Observability Pipelines with Datadog
 kind: documentation
 aliases:
   - /agent/vector_aggregation/
   - /integrations/observability_pipelines/integrate_vector_with_datadog/
+  - /observability_pipelines/integrate_vector_with_datadog/
```

Comment:
Can we change this to /observability_pipelines_worker_[text]/?

Reply:
This is an alias.